pkg/common/telemetry/names.go

```diff
@@ -263,6 +263,15 @@ const (
 	// ElapsedTime tags some duration of time.
 	ElapsedTime = "elapsed_time"
 
+	// EntryAdded is the counter key for when a entry is added to LRU cache
```

Comment:
remember as well to update https://github.com/spiffe/spire/blob/3117f7b007f1230c2107a5127f67aa3ff836bed3/doc/telemetry.md to document the new metrics being defined